app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -170,7 +170,7 @@ def parse_node(node)
                ManageIQ::Providers::Vmware::InfraManager::Vm.name]
 
       # Searching for the underlying instance for Openstack or oVirt.
-      vms = Vm.where(:uid_ems => new_result[:identity_system].downcase, :type => types)
+      vms = Vm.where(:uid_ems => new_result[:identity_system].try(:downcase), :type => types)
```

Comment:
if the try returns nil, then how will the query behave with the condition uid_ems=nil ?